rpcs3/Emu/Cell/Modules/cellSysCache.cpp

```diff
@@ -121,9 +123,12 @@ error_code cellSysCacheMount(vm::ptr<CellSysCacheParam> param)
 	auto& cache = g_fxo->get<syscache_info>();
 
 	if (!param || (param->cacheId[0] && sysutil_check_name_string(param->cacheId, 1, CELL_SYSCACHE_ID_SIZE) != 0))
-	{
 		return CELL_SYSCACHE_ERROR_PARAM;
-	}
+
+	std::unique_lock lock_hdd1(g_mp_sys_dev_hdd1.mutex, std::try_to_lock);
+
+	if (!lock_hdd1.owns_lock())
+		return CELL_EBUSY;
```

Comment:
Please do not reinvent PS3 operating system, the function is the same and this error code does not exist for it.

Reply:
Thanks for correcting.
I reverted most of that.